src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -579,20 +587,25 @@ public void TotalProcessorTime_GetNotStarted_ThrowsInvalidOperationException()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/22266", TargetFrameworkMonikers.UapNotUapAot)]
         public void TestProcessStartTime()
         {
-            TimeSpan allowedWindow = TimeSpan.FromSeconds(3);
-            DateTime testStartTime = DateTime.UtcNow;
-            using (var remote = RemoteInvoke(() => { Console.Write(Process.GetCurrentProcess().StartTime.ToUniversalTime()); return SuccessExitCode; },
-                new RemoteInvokeOptions { StartInfo = new ProcessStartInfo { RedirectStandardOutput = true } }))
-            {
-                Assert.Equal(remote.Process.StartTime, remote.Process.StartTime);
+            TimeSpan allowedWindow = TimeSpan.FromSeconds(1);
```

Comment:
This is shrinking the window substantially.  Was the original value just wildly too big, or was it ever increased due to instability of the test in CI?